<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetailsTicket</class>
 <widget class="QDialog" name="DetailsTicket">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Informations du ticket</string>
  </property>
  <widget class="QLabel" name="labelHeader">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>641</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>EasyTicket</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitre">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>521</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Titre ticket</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>59</y>
     <width>133</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="infosLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>50</number>
    </property>
    <item>
     <widget class="QLabel" name="labelID">
      <property name="text">
       <string>ID
0000</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelStatut">
      <property name="text">
       <string>Statut
Ouvert</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelDate">
      <property name="text">
       <string>Créé le
01/12/2022</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelCategorie">
      <property name="text">
       <string>Catégorie
Question</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelLogiciel">
      <property name="text">
       <string>Logiciel
Twisk</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelPrisEnCharge">
      <property name="text">
       <string>Pris en charge par
Jean Dupond
01/12/2022</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="boutonCloturer">
      <property name="text">
       <string>Clôturer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="inputReponse">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>390</y>
     <width>421</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitreReponse">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>360</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nouveau message</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boutonEnvoyer">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>410</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Envoyer</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="zoneMessages">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>521</width>
     <height>251</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>505</width>
      <height>249</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>481</width>
       <height>231</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="zoneMessagesLayout"/>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
